src/main/java/com/blibli/oss/command/autoconfigure/CommandAutoConfiguration.java

```diff
@@ -23,6 +23,9 @@
   @Bean
   @ConditionalOnMissingBean
   public CommandExecutor commandExecutor(@Autowired Validator validator) {
+    if (false) {
+      System.out.println("rubbish");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS106)